ydb/library/yql/udfs/common/python/ya.make

```diff
@@ -0,0 +1,9 @@
+# This module should not be exported under CMake since it requires Python build
+NO_BUILD_IF(STRICT EXPORT_CMAKE)
+
+RECURSE(
+    bindings
+    main_py3
+    python3_small
```

Comment:
I suggest to remove `_small` suffix from the udf name

Reply:
please keep python3_small as is. OSS versions are based on system pythons